plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -280,6 +280,9 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 		Rules: []rbacv1.PolicyRule{
 			rbacv1helpers.NewRule("list", "watch", "delete").Groups(legacyGroup).Resources("pods").RuleOrDie(),
 			rbacv1helpers.NewRule("get", "list").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+			// used for pod deletion
+			rbacv1helpers.NewRule("patch").Groups(legacyGroup).Resources("pods/status").RuleOrDie(),
```

Comment:
pod GC doesn't have permission to write to pods at all today... only add this permission if the pod condition feature is enabled

Reply:
Done, it is now protected by the feature gate flag? Should we also make an analogous change to only add "patch" to node-controller?

Reply:
> Should we also make an analogous change to only add "patch" to node-controller?

That's less of a big deal... it already had update permission, adding `patch` is not granting it more power, just an additional way to accomplish the same thing.

This line is adding write permission where it didn't exist previously, so gating makes more sense